jlm/rvsdg/region.hpp

```diff
@@ -50,13 +50,11 @@ class RegionArgument : public output
 
   ~RegionArgument() noexcept override;
 
-protected:
   RegionArgument(
       rvsdg::Region * region,
       structural_input * input,
       std::shared_ptr<const rvsdg::Type> type);
 
-public:
```

Comment:
Deleted copy and move constructors should be public.

Reply:
they are public already (everything is public now)